README.md

```diff
@@ -2,7 +2,7 @@
 # to do List
 > This is a simple html markup page
 
```

Comment:
- [ ] Kindly add a good description of this project describing this project in detail. This will improve your README and users can know all about your project just by reading your README.

Reply:
Thanks for the review, @julie-ify I have added the project description.